pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -190,6 +190,10 @@ const (
 	NodeImageFamilyWindowsServer2019CoreContainer = "WindowsServer2019CoreContainer"
 	NodeImageFamilyWindowsServer2019FullContainer = "WindowsServer2019FullContainer"
 	NodeImageFamilyWindowsServer2004CoreContainer = "WindowsServer2004CoreContainer"
+)
+
+// Deprecated `NodeAMIFamily`
+const (
 	NodeImageFamilyWindowsServer20H2CoreContainer = "WindowsServer20H2CoreContainer"
```

Comment:
need to remove this too probably

Reply:
I added this `// Deprecated NodeAMIFamily` group to use the `const`s in the validation.

Reply:
I suggest we keep this until we can remove the validation for the deprecated Windows AMIs. Normally we have a 6 month deprecation policy, how long should we wait for this?

Reply:
We should keep the consts for deprecated AMI families for at least 6 months so we can identify deprecated AMI families and point users to the deprecation notice in the returned error message.

Reply:
Sounds good 👍🏻